pkg/handlers/v1/db.go

```diff
@@ -0,0 +1,326 @@
+package v1
```

Comment:
This should be moved to a package named after the file in which the interface is defined. In this case `storage`
https://developer.atlassian.com/platform/security/guides/google-golang/service-repo/